### ISSUE TITLE
The difference between table1 and table3

### ISSUE THREAD

**Issue by WERush** [fangchangma/self-supervised-depth-completion#5]
Hi, Fangchang,

What is the difference between "Ours-RGBd" in table1 and "supervised learning " in table2? 
Specifically, "Ours-RGBd" achieves rmse=814.73 in Test set while "supervised learning" is 878.56 in Validation set. Why is the performance on Test set much higher than that on validation set?

Thanks very much.

**Comment by DecaYale** [comment]
I am also confused by the configuration differences between these tables and I find it hard to achieve the performances reported in the paper. To be more specific, our RMSE  only reaches about 870mm on the test set while yours achieves 814mm. I may have made some mistakes and I wonder what is the key to the improvement. 

Thanks.

**Comment by fangchangma** [comment]
> Why is the performance on Test set much higher than that on validation set?

We simply reported the accuracy of the same network on both validation and test sets, without any tweaking. For all our experiments on the KITTI depth completion task, the test accuracy happened to be slightly better than the validation accuracy. It is possible that the test dataset is somehow "simpler" than the validation dataset, for this particular KITTI benchmark.

**Comment by DecaYale** [comment]
Hi, 
I have noticed that the best performance in table 2 is rmse=856.754mm while the best in table 3 is rmse=878.56mm. What are the configuration differences between these two models?  

Thanks.

**Comment by fangchangma** [comment]
Hi @DecaYale. Thanks for catching this. It is probably because we forgot to update Table 3. We'll make the modifications in a later release of the paper.
